docs/ksql-clickstream-demo/non-docker-clickstream.rst

```diff
@@ -0,0 +1,364 @@
+    process is running as a daemon, so just press return again to see
+    the shell prompt.
+
+    .. code:: bash
+
+        $ <path-to-ksql>/bin/ksql-datagen  -daemon quickstart=clickstream format=json topic=clickstream maxInterval=100 iterations=500000
+
+    Your output should resemble:
+
+    .. code:: bash
+
+        Writing console output to /tmp/ksql-logs/ksql.out
+
+7.  From your terminal, create the status codes using the ksql-datagen
+    utility. This stream runs once to populate the table.
+
+    .. code:: bash
+
+        $ <path-to-ksql>/bin/ksql-datagen  quickstart=clickstream_codes format=json topic=clickstream_codes maxInterval=20 iterations=100
+
+    Your output should resemble:
+
+    .. code:: bash
+
+        200 --> ([ 200 | 'Successful' ])
+        302 --> ([ 302 | 'Redirect' ])
+        200 --> ([ 200 | 'Successful' ])
+        406 --> ([ 406 | 'Not acceptable' ])
+        ...
+
+8.  From your terminal, create a set of users using ksql-datagen
+    utility. This stream runs once to populate the table.
+
+    .. code:: bash
+
+        $ <path-to-ksql>/bin/ksql-datagen  quickstart=clickstream_users format=json topic=clickstream_users maxInterval=10 iterations=1000
+
+    Your output should resemble:
+
+    .. code:: bash
+
+        1 --> ([ 1 | 'GlenAlan_23344' | 1424796387808 | 'Curran' | 'Lalonde' | 'Palo Alto' | 'Gold' ])
+        2 --> ([ 2 | 'ArlyneW8ter' | 1433932319457 | 'Oriana' | 'Vanyard' | 'London' | 'Platinum' ])
+        3 --> ([ 3 | 'akatz1022' | 1478233258664 | 'Ferd' | 'Trice' | 'Palo Alto' | 'Platinum' ])
+        ...
+
+9.  Launch the KSQL CLI in local mode.
+
+    .. code:: bash
+
+        $ <path-to-ksql>/bin/ksql-cli local
+
+    You should see the KSQL CLI welcome screen.
+
+    .. code:: bash
+
+                           ======================================
+                           =      _  __ _____  ____  _          =
+                           =     | |/ // ____|/ __ \| |         =
+                           =     | ' /| (___ | |  | | |         =
+                           =     |  <  \___ \| |  | | |         =
+                           =     | . \ ____) | |__| | |____     =
+                           =     |_|\_\_____/ \___\_\______|    =
+                           =                                    =
+                           =   Streaming SQL Engine for Kafka   =
+        Copyright 2017 Confluent Inc.                         
+
+        CLI v0.1, Server v0.1 located at http://localhost:9098
+
+        Having trouble? Type 'help' (case-insensitive) for a rundown of how things work!
+
+        ksql>
+
+10. From the the KSQL CLI, load the ``clickstream.sql`` schema file that
+    will run the demo app.
+
+    **Important:** Before running this step, you must have already run
+    ksql-datagen utility to create the clickstream data, status codes,
+    and set of users.
+
+    ::
+
+        ksql> run script 'ksql-clickstream-demo/demo/clickstream-schema.sql';
```

Comment:
Syntax nit: I think we want to standardize on all-caps for SQL keywords, so `run script ...` would need to be `RUN SCRIPT ...`?